### ISSUE TITLE
The training loss 

### ISSUE THREAD

**Issue by eeewhe** [IvonaTau/emotionaldan#2]
Hi, can you share your training loss variation? It seems my training loss is not normal...
**Epoch (1/20), batch (0/4495), 0.845s, batch err: 5.754796981811523
Epoch (1/20), batch (100/4495), 0.819s, batch err: 19.57297134399414
Epoch (1/20), batch (200/4495), 0.827s, batch err: 1.2522774934768677
Epoch (1/20), batch (300/4495), 0.823s, batch err: 1.128464937210083
Epoch (1/20), batch (400/4495), 0.822s, batch err: 1.730790376663208
Epoch (1/20), batch (500/4495), 0.826s, batch err: 1.2757847309112549
Epoch (1/20), batch (600/4495), 0.826s, batch err: 1.1355090141296387
Epoch (1/20), batch (700/4495), 0.822s, batch err: 1.235153317451477
Epoch (1/20), batch (800/4495), 0.830s, batch err: 1.315584421157837
Epoch (1/20), batch (900/4495), 0.825s, batch err: 1.122027039527893
Epoch (1/20), batch (1000/4495), 0.821s, batch err: 1.2424519062042236
Epoch (1/20), batch (1100/4495), 0.823s, batch err: 8.125507354736328
Epoch (1/20), batch (1200/4495), 0.823s, batch err: 2.4139723777770996
Epoch (1/20), batch (1300/4495), 0.826s, batch err: 2.4237818717956543
Epoch (1/20), batch (1400/4495), 0.818s, batch err: 1.1923820972442627
Epoch (1/20), batch (1500/4495), 0.833s, batch err: 1.8711756467819214
Epoch (1/20), batch (1600/4495), 0.824s, batch err: 1.2039916515350342
Epoch (1/20), batch (1700/4495), 0.823s, batch err: 1.0933177471160889
Epoch (1/20), batch (1800/4495), 0.821s, batch err: 10.37353515625
Epoch (1/20), batch (1900/4495), 0.841s, batch err: 1.202012538909912
Epoch (1/20), batch (2000/4495), 0.831s, batch err: 2.8420190811157227
Epoch (1/20), batch (2100/4495), 0.832s, batch err: 113.91435241699219
Epoch (1/20), batch (2200/4495), 0.834s, batch err: 21.43947982788086
Epoch (1/20), batch (2300/4495), 0.820s, batch err: 156.9434051513672
Epoch (1/20), batch (2400/4495), 0.836s, batch err: 3.8966124057769775
Epoch (1/20), batch (2500/4495), 0.822s, batch err: 1.2476005554199219
Epoch (1/20), batch (2600/4495), 0.849s, batch err: 2.525386333465576
Epoch (1/20), batch (2700/4495), 0.835s, batch err: 1.8467141389846802
Epoch (1/20), batch (2800/4495), 0.823s, batch err: 1.0108225345611572
Epoch (1/20), batch (2900/4495), 0.822s, batch err: 1.1918048858642578
Epoch (1/20), batch (3000/4495), 0.828s, batch err: 1.3513712882995605
Epoch (1/20), batch (3100/4495), 0.820s, batch err: 17.738948822021484
Epoch (1/20), batch (3200/4495), 0.835s, batch err: 1.3260831832885742
Epoch (1/20), batch (3300/4495), 0.824s, batch err: 1.2585244178771973
Epoch (1/20), batch (3400/4495), 0.841s, batch err: 1.078512191772461
Epoch (1/20), batch (3500/4495), 0.821s, batch err: 128.75778198242188
Epoch (1/20), batch (3600/4495), 0.828s, batch err: 177.14918518066406
Epoch (1/20), batch (3700/4495), 0.821s, batch err: 1.050371766090393
Epoch (1/20), batch (3800/4495), 0.824s, batch err: 3.4665944576263428
Epoch (1/20), batch (3900/4495), 0.847s, batch err: 1.1717772483825684
Epoch (1/20), batch (4000/4495), 0.826s, batch err: 1.45053231716156
Epoch (1/20), batch (4100/4495), 0.825s, batch err: 4.0133376121521
Epoch (1/20), batch (4200/4495), 0.824s, batch err: 25.501022338867188
Epoch (1/20), batch (4300/4495), 0.822s, batch err: 1.2257040739059448
Epoch (1/20), batch (4400/4495), 0.823s, batch err: 1.6352043151855469
Epoch (1/20), val err: 40.76654824756441**

**Comment by IvonaTau** [comment]
What batch size are you using?
